api/topology/src/main/java/quarks/topology/TStream.java

```diff
@@ -443,7 +443,7 @@ else if (WARNING.equals(lr.getLevel()))
     Set<String> getTags(); 
     
     /**
-     * Add an alias for the stream.
+     * Set an alias for the stream.
      * <p>
      * The alias must be unique within the topology.
```

Comment:
I think it has to be that restrictive for things to remain sane.  What you say is true, but a TStream.alias() user has no idea or control over what "types" may be registered by the runtime or others using an alias so they can't predict or otherwise avoid collisions, right?   

E.g., maybe some day poll() and map() register some mbean with the same type, say, some logging mbean or such?

At a higher level I'm not really getting it wrt picking "type" or alias names, or "ids" for that matter in order to generally avoid collisions since the ControlService is documented as expecting to be used by application code as well as the runtime.  Seems like minimally more doc is needed in ControlService for conventions for these.  And right now the runtime uses unadorned values like "job" and "periodic" for types - what's to prevent an ignorant user for picking those to use for their own application specific controls?